wifiphisher/data/wifiphiser-os-initial-requests

```diff
@@ -0,0 +1,9 @@
+Android|google.com/generate_204
+Android|connectivitycheck.gstatic.com/generate_204
+Android|connectivitycheck.android.com/generate_204
+Android|gstatic.com/generate_204
+Apple|captive.apple.com/hotspot-detect.html
```

Comment:
I wonder if this is for both iOS devices and MacBooks.

Reply:
i have found the captive.apple.com/hotspot-detect.html referenced in the web by both ios10 devices and MacBooks

Reply:
apple.com/library/test/success.html alredy exists in line 6, i do not know if its only for ios devices or MacOS as well

Reply:
also urls like: akamaitechnologies.com which is a CDN network can be accessed by other OS as well and i think should not be taken into account